### ISSUE TITLE
Does not handle quoted keys in map directive

### ISSUE THREAD

**Issue by nicksellen** [tmont/nginx-conf#10]
The `map` directive (http://nginx.org/en/docs/http/ngx_http_map_module.html) allows keys to be quoted with single or double quoted strings and this breaks the library with `Found a string but expected a directive` error. 

An example for the empty string case (http://nginx.org/en/docs/http/websocket.html):

```
map $http_upgrade $connection_upgrade {
  default upgrade;
  ''      close;
}
```

I can write a fix if you'd like?

**Comment by tmont** [comment]
This should be fixed in `nginx-conf@1.2.0`. It's been published to NPM.

Note that you'll have to access the properties via the `[]` syntax:

`nginx.conf`: 
```nginx
http {
  map $http_upgrade $connection_upgrade {
    default upgrade;
    ''      close;
  }
}
```

`nginx.js`:
```javascript
console.log(file.nginx.http.map._value); // $http_upgrade $connection_upgrade
console.log(file.nginx.http.map.default._value); // upgrade
console.log(file.nginx.http.map['\'\'']._value); // close
```

Thanks for the bug report!

**Comment by nicksellen** [comment]
Great, works perfectly :)

Thanks for the speedy fix!
